book/src/user/target-tier-policies.md

```diff
@@ -0,0 +1,243 @@
+evaluating the potential demotion or removal of a platform that has been part
+of a stable release. The promotion or demotion of a platform will not generally
+affect existing stable releases, only current development and future releases.
+
+In this policy, the words "must" and "must not" specify absolute requirements
+that a platform must meet to qualify for a tier. The words "should" and "should
+not" specify requirements that apply in almost all cases, but for which the
+Zebra team may grant an exception for good reason. The word "may" indicates
+something entirely optional, and does not indicate guidance or recommendations.
+This language is based on [IETF RFC 2119](https://tools.ietf.org/html/rfc2119).
+
+## Tier 3 platform policy
+
+At this tier, the Zebra project provides no official support for a platform, so
+we place minimal requirements on the introduction of platforms.
+
+- A tier 3 platform must have a designated developer or developers (the "platform
+  maintainers") on record to be CCed when issues arise regarding the platform.
+  (The mechanism to track and CC such developers may evolve over time.)
+- Target names should not introduce undue confusion or ambiguity unless
+  absolutely necessary to maintain ecosystem compatibility. For example, if
+  the name of the platform makes people extremely likely to form incorrect
+  beliefs about what it targets, the name should be changed or augmented to
+  disambiguate it.
+- Tier 3 platforms must not impose burden on the authors of pull requests, or
+  other developers in the community, to maintain the platform. In particular,
+  do not post comments (automated or manual) on a PR that derail or suggest a
+  block on the PR based on a tier 3 platform. Do not send automated messages or
+  notifications (via any medium, including via `@`) to a PR author or others
+  involved with a PR regarding a tier 3 platform, unless they have opted into
+  such messages.
+- Patches adding or updating tier 3 platforms must not break any existing tier 2
+  or tier 1 platform, and must not knowingly break another tier 3 platform
+  without approval of either the Zebra team of the other tier 3 platform.
+
+If a tier 3 platform stops meeting these requirements, or the platform maintainers
+no longer have interest or time, or the platform shows no signs of activity and
+has not built for some time, or removing the platform would improve the quality
+of the Zebra codebase, we may post a PR to remove it; any such PR will be CCed
+to the platform maintainers (and potentially other people who have previously
+worked on the platform), to check potential interest in improving the situation.
+
+## Tier 2 platform policy
+
+At this tier, the Zebra project guarantees that a platform builds, and will reject
+patches that fail to build on a platform. Thus, we place requirements that ensure
+the platform will not block forward progress of the Zebra project.
```

Comment:
Should tier 2 Zebra platforms require a specific Rust compiler tier?
Do we want to require Rust tier 1 or 2?

Reply:
This could (in the future) make our Tiers unpredictable, not making this document a single source of truth. But, at the same time, I kind of agree that this should be expressed somehow.

Isn't the `supported-platforms.md` clarifying this with the Rust versions column on each Tier?

Reply:
The Rust versions are different from the Rust platform tiers.

The same Rust stable release version is guaranteed to work on tier 1 platforms, and build on tier 2 platforms. So I guess we have to require tier 1, because a working Rust compiler is required to build Zebra.

Reply:
I added some suggested working in tier 2 to cover this.